src/components/Combobox/Combobox.js

```diff
@@ -0,0 +1,303 @@
+// import cx from 'classnames';
```

Comment:
Should we remove this file for this PR? Same for the corresponding test and styles.

Reply:
Depending on how much I can chip away at ComboBox before the sprint ends, I might just remove those files altogether and save them for another PR 😄

Reply:
@joshblack Yeah this is something I had in mind - Great if this PR can be splitted, one for `MultiSelect` and one for filtering select.